iw/filter_gui.py

```diff
@@ -924,8 +926,6 @@ def _active(info):
             # We use a copy here, so as to not modify the original udev info
             # dict as that would break NameCache matching
             data = mpath[0].copy()
-            # This will swap the device name (eg: sda) for the mpath name (eg: mpatha)
-            data["name"] = udev_device_get_multipath_name(mpath[0])
```

Comment:
Okay, I think I understand as much as I want to now. If you're not changing the name you also shouldn't be making (and using) a copy of the actual device data.